.github/workflows/blank.yml

```diff
@@ -26,6 +26,7 @@ jobs:
       - uses: actions/checkout@v3
       - run: "apt update"
       - run: "apt install -y python3-venv"
-      - run: "python3 -m venv venv"
+      - run: "python3 -m venv venv --system-site-packages"
       - run: "./venv/bin/pip install -r requirements.txt"
       - run: "${GITHUB_WORKSPACE}/spec/test-spec.sh"
+      - run: "./venv/bin/python3 manage.py migrate"
```

Comment:
Since this is just python code, I can put the code everywhere and run it. I've put a new virtual environment inside the current working directory (of the container) where I am installing the requirements to. The reason to use a virtual environment instead of putting it inside the system's site packages is to properly isolate the dependencies of each (python-)program on the system.

I've also discussed this with @kodebach today during the tutoring lesson.

Nonetheless I've improved the README.md to actually have a step-by-step-guide on how to install this from scratch (without GitHub Actions, ...). This should be of great use for the person ending up doing HW3 with this project 😃